superset/assets/spec/javascripts/modules/dates_spec.js

```diff
@@ -0,0 +1,80 @@
+import { it, describe } from 'mocha';
+import { expect } from 'chai';
+import {
+  tickMultiFormat,
+  formatDate,
+  timeFormatFactory,
+  fDuration,
+  now,
+  epochTimeXHoursAgo,
+  epochTimeXDaysAgo,
+  epochTimeXYearsAgo,
+ } from '../../../javascripts/modules/dates';
+
+describe('tickMultiFormat', () => {
+  it('is a function', () => {
+    assert.isFunction(tickMultiFormat);
+  });
+});
+
+describe('formatDate', () => {
+  it('is a function', () => {
+    assert.isFunction(formatDate);
+  });
+});
+
+describe('timeFormatFactory', () => {
+  it('is a function', () => {
+    assert.isFunction(timeFormatFactory);
+  });
+});
+
+describe('fDuration', () => {
+  it('is a function', () => {
+    assert.isFunction(fDuration);
+  });
+
+  it('returns a string', () => {
+    expect(fDuration(new Date(), new Date())).to.be.a('string');
```

Comment:
Besides return type, should test the return value of fDuration function is the diff of first 2 params.